opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/FragmentBaseActivity.java

```diff
@@ -73,13 +174,28 @@ protected void onCreate(Bundle savedInstanceState) {
                 titleTextView.setText(title);
             }
             INDICATOR_CODE = bundle.getString(INDICATOR_CODE);
+
+            String report_date = bundle.getString(GizConstants.ReportParametersHelper.REPORT_DATE);
+            String communityName = bundle.getString(GizConstants.ReportParametersHelper.COMMUNITY);
             String fragmentName = bundle.getString(DISPLAY_FRAGMENT);
+            String name = getRequestedFragmentName(fragmentName);
+
+            //Format Subject Date
+            if (report_date != null) {
+                DateFormat df = new SimpleDateFormat("dd MMM yyyy");
+                Date date = null;
+                try {
+                    date = df.parse(report_date);
+                } catch (ParseException e) {
+                    e.printStackTrace();
+                }
+                emailSubject = String.format("%s - %s - %s", name, new SimpleDateFormat("dd/MM/yyyy hh:mm:ss").format(date), communityName);
+                emailAttachmentName = String.format("%s-%s-%s", name, new SimpleDateFormat("ddMMyyyy hh:mm:ss").format(date), communityName);
```

Comment:
Should these two have the same value?

Reply:
No, The Requirement was to have them as above.